### PR TITLE
Add default config set for tuning

### PR DESCRIPTION
**Pull request by yiliu30**
## Type of Change

Feature, implement the https://github.com/intel/neural-compressor/discussions/1445#discussioncomment-8104344
API changed or not: YES

## Description

- API change
```diff
+ get_all_config_set()
- TuningConfig(quant_configs=...)
+ TuningConfig(config_set=get_all_config_set(), max_trials=4)
```

- Usage
```python
from neural_compressor.torch import TuningConfig, autotune, get_all_config_set
tune_config = TuningConfig(config_set=get_all_config_set(), max_trials=4)
```

- FWK owner TODO
``` python
class AlgoConfig(BaseConfig):
    ...
    # fwk owner needs to implement the `get_config_set_for_tuning` for each algo config.
    @classmethod
    def get_config_set_for_tuning(cls):
        ...
```

## Test
Pre-CI

## Dependency Change?
None

### CODE REVIEW

File: neural_compressor/onnxrt/quantization/config.py [hunk 1]
```diff
@@ -144,6 +144,11 @@ def get_model_info(model: Union[onnx.ModelProto, Path, str]) -> List[Tuple[str,
         logger.debug(f"Get model info: {filter_result}")
         return filter_result
 
+    @classmethod
+    def get_config_set_for_tuning(cls) -> Union[None, "RTNConfig", List["RTNConfig"]]:  # pragma: no cover
```
Comment (yiliu30) [inline neural_compressor/onnxrt/quantization/config.py:148]: @mengniwang95  @yuwenzho, we need to update it after the ORT enables the auto-tune.

File: neural_compressor/tensorflow/quantization/config.py [hunk 2]
```diff
@@ -103,6 +103,13 @@ def register_supported_configs(cls) -> List[OperatorConfig]:
         supported_configs.append(OperatorConfig(config=static_quant_config, operators=operators))
         cls.supported_configs = supported_configs
 
+    @classmethod
+    def get_config_set_for_tuning(
+        cls,
+    ) -> Union[None, "StaticQuantConfig", List["StaticQuantConfig"]]:  # pragma: no cover
```
Comment (yiliu30) [inline neural_compressor/tensorflow/quantization/config.py:109]: @zehao-intel , we need to update it after the TF enables the auto-tune.

File: neural_compressor/torch/quantization/config.py [hunk 3]
```diff
@@ -145,6 +153,11 @@ def get_model_info(model: torch.nn.Module) -> List[Tuple[str, Callable]]:
         logger.debug(f"Get model info: {filter_result}")
         return filter_result
 
+    @classmethod
+    def get_config_set_for_tuning(cls) -> Union[None, "RTNConfig", List["RTNConfig"]]:
+        # TODO fwk owner needs to update it.
+        return RTNConfig(weight_bits=[4, 6])
```
Comment (yiliu30) [inline neural_compressor/torch/quantization/config.py:159]: Hi @xin3he @violetch24,  we should update the argument values before announcing the auto-tune API.

**Review by yuwenzho — APPROVED** [review]

**Review by mengniwang95 — APPROVED** [review]

File: neural_compressor/tensorflow/quantization/config.py (hunk 2 above)
Comment (zehao-intel) [reply neural_compressor/tensorflow/quantization/config.py:109]: Get it, thanks.

**Review by zehao-intel — APPROVED** [review]

**Pull request merged**